src/query/src/metrics.rs

```diff
@@ -18,3 +18,6 @@ pub static METRIC_OPTIMIZE_LOGICAL_ELAPSED: &str = "query.optimize_logicalplan_e
 pub static METRIC_OPTIMIZE_PHYSICAL_ELAPSED: &str = "query.optimize_physicalplan_elapsed";
 pub static METRIC_CREATE_PHYSICAL_ELAPSED: &str = "query.create_physicalplan_elapsed";
 pub static METRIC_EXEC_PLAN_ELAPSED: &str = "query.execute_plan_elapsed";
+pub static METRIC_MERGE_SCAN_POLL_ELAPSED: &str = "query.merge_scan.poll_elapsed";
+pub static METRIC_MERGE_SCAN_REGIONS: &str = "query.merge_scan.regions";
+pub static METRIC_MERGE_SCAN_ERRORS_TOTAL: &str = "query.merge_scan.errors_total";
```

Comment:
I think the `_total` postfix is unnecessary.

Reply:
`_TOTAL` postfix is a prometheus metric convention for counter